__tests__/unit/core-forger/manager.test.ts

```diff
@@ -115,6 +115,20 @@ describe("Forger Manager", () => {
             });
 
             expect(forgeManager.client.broadcastBlock).not.toHaveBeenCalled();
+
+            jest.resetAllMocks();
+
+            jest.spyOn(Crypto.Slots, "getTimeInMsUntilNextSlot").mockReturnValueOnce(2000);
+            jest.spyOn(Crypto.Slots, "getSlotNumber").mockImplementation(epoch => {
+                return epoch ? 1 : 2;
```

Comment:
What about `s/epoch/timestamp/`?
And one or both of `(timestamp?) =>...` or `return timestamp !== undefined ? 1 : 2;` to hint that we are testing for `undefined`, not for `0`